components/log-viewer-webui/server/src/app.test.js

```diff
@@ -5,20 +5,22 @@ import app from "./app.js";
 
 
 test("Tests the example routes", async (t) => {
-    const server = await app();
+    process.env.NODE_ENV = "test";
```

Comment:
Instead of forcibly setting the environment, maybe we should assert that it's "test"?

Reply:
In fact this NODE_ENV is unset if we don't have the line. However, I do agree the setting of it should be done in the package.json (we should specify it when running `tap`).